### PR TITLE
Fix iconv

### PR DESCRIPTION
**Pull request by 3noch**

### CODE REVIEW

**Comment by 3noch** [comment]
Hmm this looks more different than it should I think.

File: libiconv-wasm32.patch [hunk 1]
```diff
@@ -1,6 +1,6 @@
-diff -ruN libiconv-1.15/build-aux/config.sub libiconv-1.15-patched/build-aux/config.sub
---- libiconv-1.15/build-aux/config.sub	2017-01-02 09:02:47.000000000 +1000
-+++ libiconv-1.15-patched/build-aux/config.sub	2017-11-22 08:38:36.995036311 +1000
```
Comment (ElvishJerricco) [inline libiconv-wasm32.patch]: Yea it'd be nice if the patch directory names could have stayed the same. i.e. `libiconv-1.15` and `libiconv-1.15-patched`.

File: libiconv-wasm32.patch [hunk 2]
```diff
@@ -19,9 +19,37 @@ diff -ruN libiconv-1.15/build-aux/config.sub libiconv-1.15-patched/build-aux/con
...
-+++ libiconv-1.15-patched/libcharset/build-aux/config.sub	2017-11-22 08:52:03.504787774 +1000
+diff -r -u libiconv-1.15/build-aux/ltmain.sh libiconv-1.15-mine/build-aux/ltmain.sh
+--- libiconv-1.15/build-aux/ltmain.sh	2015-06-28 08:55:35.000000000 -0400
++++ libiconv-1.15-mine/build-aux/ltmain.sh	2018-05-01 00:15:35.011163337 -0400
+@@ -7459,8 +7459,8 @@
+     else
+       shlib_search_path=
+     fi
+-    eval sys_lib_search_path=\"$sys_lib_search_path_spec\"
+-    eval sys_lib_dlsearch_path=\"$sys_lib_dlsearch_path_spec\"
```
Comment (ElvishJerricco) [inline libiconv-wasm32.patch]: This is odd. Why would this need to change?

File: libiconv-wasm32.patch [hunk 3]
```diff
@@ -40,3 +68,17 @@ diff -ruN libiconv-1.15/libcharset/build-aux/config.sub libiconv-1.15-patched/li
...
  	*)
+diff -r -u libiconv-1.15/libcharset/build-aux/ltmain.sh libiconv-1.15-mine/libcharset/build-aux/ltmain.sh
+--- libiconv-1.15/libcharset/build-aux/ltmain.sh	2015-06-28 08:55:35.000000000 -0400
++++ libiconv-1.15-mine/libcharset/build-aux/ltmain.sh	2018-05-01 00:15:35.052164642 -0400
+@@ -7459,8 +7459,8 @@
+     else
+       shlib_search_path=
+     fi
+-    eval sys_lib_search_path=\"$sys_lib_search_path_spec\"
+-    eval sys_lib_dlsearch_path=\"$sys_lib_dlsearch_path_spec\"
```
Comment (ElvishJerricco) [inline libiconv-wasm32.patch]: Same question

**Review by ElvishJerricco — CHANGES_REQUESTED** [review]

File: libiconv-wasm32.patch (hunk 1 above)
Comment (3noch) [reply libiconv-wasm32.patch]: I'll fix.

**Comment by 3noch** [comment]
I'll try this again but more systematically and figure out why the patch is...different.

**Comment by ElvishJerricco** [comment]
i believe I've fixed this.

**Pull request merged**